frontend/src/components/Postcard.jsx

```diff
@@ -26,37 +27,37 @@ function Postcard() {
             />
           </div>
         </div>
-        <div className="contentPost">
-          {/* section with name and information about the post */}
-          <div className="headpost">
-            <div>Pseudo</div>
-            <div>TeamName</div>
-            <div>postTime</div>
-          </div>
+        <div className="contentPostContainer">
+          <div className="contentPost">
+            {/* section with name and information about the post */}
+            <div className="headpost">
+              <div>Pseudo</div>
+              <div>TeamName</div>
+              <div>
+                <Moment format="L" date={props.date} />
+                <Moment format="h:mm" date={props.date} />
+              </div>
+            </div>
 
-          {/* section with the content of the post*/}
-          <div className="contentpost">
-            <div className=" mediaContainer">
+            {/* section with the content of the post*/}
+            <div className="contentpost">
+              <div className="mediaContainer">
+                {/* section with the media*/}
+                <img
+                  className="postmedia"
+                  src="https://via.placeholder.com/500x300"
```

Comment:
Either there is a content and you display it, or there is no content and you don't display the whole block, but I don't want any more placeholders (_Personal opinion: plus, they're ugly as fck !_)

Reply:
can we do it in a future PR :D ?

Reply:
<Joke>To replace mine, I'll rip your eyes if I see placeholders on next Friday :D</Joke>
Ok, you may do this on a future PR